editor/src/clj/editor/lsp/server.clj

```diff
@@ -67,13 +68,21 @@
 (extend-protocol Launcher
   Map
   (launch [{:keys [command]} directory]
-    (.start
-      (doto (ProcessBuilder. ^"[Ljava.lang.String;" (into-array String command))
-        (.redirectError ProcessBuilder$Redirect/INHERIT)
-        (.directory directory)))))
+    {:pre [(vector? command)]}
+    ;; We need to resolve command in addition to setting ProcessBuilder's directory,
+    ;; since the latter only affects the current directory of the spawned process,
+    ;; and does not affect executable resolution
```

Comment:
I have no idea how relative paths worked on macOS, but on Windows, I needed to specify the full path for the process to start.